data/hai/hai ships.txt

```diff
@@ -177,6 +177,62 @@ ship "Centipede" "Centipede (Refugees)"
 
 
 
+ship "Cicada"
+	sprite "ship/hai cicada"
+	thumbnail "thumbnail/hai cicada"
+	attributes
+		category "Transport"
+		"cost" 5850000
+		"shields" 4900
+		"hull" 2800
+		"required crew" 7
+		"bunks" 51
+		"mass" 240
+		"drag" 4.7
+		"heat dissipation" .75
+		"fuel capacity" 600
```

Comment:
```suggestion
		"cost" 3850000
		"shields" 5800
		"hull" 1700
		"required crew" 8
		"bunks" 51
		"mass" 300
		"drag" 4.7
		"heat dissipation" .75
		"fuel capacity" 500
```

To summarize the discussion, the changes proposed were something like this. 
Drop the cost down to a more reasonable level. 
Drop the hull and increase shields to better fit with Hai's focus on shielding. EHP decreased by 200 overall.
Increase mass and drag a bit to justify the relatively high cargo space and bunks (for its size) 
Drop fuel capacity. 600 is far more than it needs.